src/styles/StyleUtils.js

```diff
@@ -42,6 +43,19 @@ function getAvatarStyle(size) {
     };
 }
 
+/**
+ * Return the border style if avatar is SVG
+ *
+ * @param {Boolean} isSVG
+ * @returns {Object}
```

Comment:
We're saying return's an Object but also passing `[]`

Reply:
That was a mistake, nice catch!